app.py

```diff
@@ -459,15 +462,16 @@ def make_traces(base_data, biden_data, y, title):
             """
             **Note:** This project builds on TK paper by Pomerleau. 
             Data for this project are generated using the open-source 
-            [Cost-of-Capital-Calculator](https://github.com/PSLmodels/Cost-of-Capital-Calculator) 
-            and [Tax-Calculator](https://github.com/PSLmodels/Tax-Calculator) projects. 
+            <a href="https://github.com/PSLmodels/Cost-of-Capital-Calculator" children="Cost-of-Capital-Calculator" target="blank"/>
+            and <a href="https://github.com/PSLmodels/Tax-Calculator" children="Tax-Calculator" target="blank"/> projects. 
             The code that modifies the underlying models to produce these estimates
-            can be found [here](https://github.com/kpomerleau/Cost-of-Capital-Calculator/tree/Tests) 
-            and [here](https://github.com/erinmelly/Tax-Calculator/tree/Biden).
+            can be found <a href="https://github.com/kpomerleau/Cost-of-Capital-Calculator/tree/Tests" children="here" target="blank"/>
+            and <a href="https://github.com/erinmelly/Tax-Calculator/tree/Biden" children="here" target="blank" />.
             The code that powers this data visualization can be found
-            [here](https://github.com/Peter-Metz/ccc-widget).
+            <a href="https://github.com/Peter-Metz/ccc-widget" children="here" target="blank" />.
```

Comment:
@Peter-Metz I think we discussed these issues in the chat, but this was the only way I was able to get the links to open in a new tab.